addons/addon-base-rest-api/packages/base-controllers/lib/users-controller.js

```diff
@@ -91,30 +64,16 @@ async function configure(context) {
     }),
   );
 
+  // TODO: Do we want to support changing user passwords? Currently this API is only supported for internal Auth
```

Comment:
We can keep this API commented for now. 

For updating user passwords from the native user pool, we'll need to implement [admin-set-user-password](https://docs.aws.amazon.com/cli/latest/reference/cognito-idp/admin-set-user-password.html) later, if needed.